src/main/java/be/seeseemelk/mockbukkit/ServerSettingsMock.java

```diff
@@ -0,0 +1,157 @@
+	private int ticksPerWaterAmbientSpawns = 1;
+	private int ticksPerAmbientSpawns = 1;
+
+	public boolean isAllowEnd() {
+		return allowEnd;
+	}
+
+	public void setAllowEnd(boolean allowEnd) {
+		this.allowEnd = allowEnd;
+	}
+
+	public String getUpdateFolder() {
+		return updateFolder;
+	}
+
+	public void setUpdateFolder(String updateFolder) {
+		this.updateFolder = updateFolder;
+	}
+
+	public File getWorldContainer() {
+		return worldContainer;
+	}
+
+	public void setWorldContainer(File worldContainer) {
+		this.worldContainer = worldContainer;
+	}
+
+	public long getConnectionThrottle() {
+		return connectionThrottle;
+	}
+
+	public void setConnectionThrottle(long connectionThrottle) {
+		this.connectionThrottle = connectionThrottle;
+	}
+
+	public String getShutdownMessage() {
+		return shutdownMessage;
+	}
+
+	public void setShutdownMessage(String shutdownMessage) {
+		this.shutdownMessage = shutdownMessage;
+	}
+
+	public int getMonsterSpawnLimit() {
+		return monsterSpawnLimit;
+	}
+
+	public void setMonsterSpawnLimit(int monsterSpawnLimit) {
+		this.monsterSpawnLimit = monsterSpawnLimit;
+	}
+
+	public int getAnimalSpawnLimit() {
+		return animalSpawnLimit;
+	}
+
+	public void setAnimalSpawnLimit(int animalSpawnLimit) {
+		this.animalSpawnLimit = animalSpawnLimit;
+	}
+
+	public int getWaterAnimalSpawnLimit() {
+		return waterAnimalSpawnLimit;
+	}
+
+	public void setWaterAnimalSpawnLimit(int waterAnimalSpawnLimit) {
+		this.waterAnimalSpawnLimit = waterAnimalSpawnLimit;
+	}
+
+	public int getWaterAmbientSpawnLimit() {
+		return waterAmbientSpawnLimit;
+	}
+
+	public void setWaterAmbientSpawnLimit(int waterAmbientSpawnLimit) {
+		this.waterAmbientSpawnLimit = waterAmbientSpawnLimit;
+	}
+
+	public int getAmbientSpawnLimit() {
+		return ambientSpawnLimit;
+	}
+
+	public void setAmbientSpawnLimit(int ambientSpawnLimit) {
+		this.ambientSpawnLimit = ambientSpawnLimit;
+	}
+
+	public int getTicksPerAnimalSpawns() {
+		return ticksPerAnimalSpawns;
+	}
+
+	public void setTicksPerAnimalSpawns(int ticksPerAnimalSpawns) {
+		this.ticksPerAnimalSpawns = ticksPerAnimalSpawns;
+	}
+
+	public int getTicksPerMonsterSpawns() {
+		return ticksPerMonsterSpawns;
+	}
+
+	public void setTicksPerMonsterSpawns(int ticksPerMonsterSpawns) {
+		this.ticksPerMonsterSpawns = ticksPerMonsterSpawns;
+	}
+
+	public int getTicksPerWaterSpawns() {
+		return ticksPerWaterSpawns;
+	}
+
+	public void setTicksPerWaterSpawns(int ticksPerWaterSpawns) {
+		this.ticksPerWaterSpawns = ticksPerWaterSpawns;
+	}
+
+	public int getTicksPerWaterAmbientSpawns() {
+		return ticksPerWaterAmbientSpawns;
+	}
+
+	public void setTicksPerWaterAmbientSpawns(int ticksPerWaterAmbientSpawns) {
+		this.ticksPerWaterAmbientSpawns = ticksPerWaterAmbientSpawns;
+	}
+
+	public int getTicksPerAmbientSpawns() {
+		return ticksPerAmbientSpawns;
+	}
+
+	public void setTicksPerAmbientSpawns(int ticksPerAmbientSpawns) {
+		this.ticksPerAmbientSpawns = ticksPerAmbientSpawns;
+	}
```

Comment:
Can you please update these methods to follow the bracket style of MockBukkit?

Also some parameter validation would be very much welcome.
or any non-primitives the parameters should have a `@NotNull` annotation and best would also be an additional
`Validate.notNull(x, "... must not be null");` check in the first line of the method.
For primitives, such as ints a simple `Validate.isTrue(x > 0), "... cannot be negative");` would be awesome too.

But we can always add these later on ourselves, just wanted to make you aware of these practices.
Those are also some examples, we would likely need to double-check with the bukkit.yml documentation to find out which values are valid and which are not.

Reply:
changed braket style
added @NotNull check for non-primitive getters.